SEED-platform/seed
# Fix SEED to work with updated ESPM API, remove SEED ESPM API v2.1

#### Any background context you want to provide?
In the last couple of days, ESPM tests have been failing. Looking at the network calls in ESPM, there appear to be some changes to the request URLs and return types.

#### What's this PR do?
* The list of reports is now returned as JSON instead of string-encoded JSON. This is a great improvement on the ESPM side, but it only pertains to this endpoint. 
* The download report no longer uses the name of the report, just the template ID. Again, another great ESPM improvement.

#### How should this be manually tested?
* Unit tests
* Connect to the SEED City Test account and verify that content downloads and loads

#### What are the relevant tickets?
#3708 

#### Screenshots (if appropriate)
<img width="747" alt="image" src="https://user-images.githubusercontent.com/1907354/207927247-31a877bd-d645-449d-a3c4-de4ee5ba641b.png">
